Editor/Unity.AddressableImporter.Editor.asmdef

```diff
@@ -2,9 +2,9 @@
     "name": "Unity.AddressableImporter.Editor",
     "references": [
         "Unity.Addressables",
-        "Unity.Addressables.Editor"
+        "Unity.Addressables.Editor",
+        "Unity.AddressableImporter.Runtime"
```

Comment:
This line shall be removed due to the `Unity.AddressableImporter.Runtime` asmdef has been removed (reverted).